app/frontend/src/features/communities/CommunityPage/DiscussionCard.tsx

```diff
@@ -95,37 +82,9 @@ export default function DiscussionCard({
             <Typography variant="h2" component="h3">
               {discussion.title}
             </Typography>
-            <Typography variant="body1">{textTruncated}</Typography>
-            <div className={classes.replies}>
-              {(thread?.pages.length ?? 0) > 0 ? (
```

Comment:
The current cards don't show any replies so I just got rid of them to match the designs.